metarParser-parsers/src/test/java/io/github/mivek/command/metar/MetarCommandArchitectureTest.java

```diff
@@ -0,0 +1,28 @@
+package io.github.mivek.command.metar;
+
+import com.tngtech.archunit.core.importer.ImportOption;
+import com.tngtech.archunit.junit.AnalyzeClasses;
+import com.tngtech.archunit.junit.ArchTest;
+import com.tngtech.archunit.junit.ArchUnitRunner;
+import com.tngtech.archunit.lang.ArchRule;
+import org.junit.runner.RunWith;
+
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.classes;
+import static com.tngtech.archunit.lang.syntax.ArchRuleDefinition.noClasses;
+
+/**
+ * @author mivek
+ */
+@RunWith(ArchUnitRunner.class)
+@AnalyzeClasses(packages = {"io.github.mivek.command.metar"}, importOptions = { ImportOption.DoNotIncludeTests.class })
+public class MetarCommandArchitectureTest {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'MetarCommandArchitectureTest' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/mivek/MetarParser/pullRequest?prid=5569897)